api/bases/neutron.openstack.org_neutronapis.yaml

```diff
@@ -2052,6 +2052,12 @@ spec:
                 default: memcached
                 description: Memcached instance name.
                 type: string
+              ml2Drivers:
```

Comment:
shouldn't we name it like e.g. "ml2MechanismDrivers" as there are different types of drivers in ML2 plugin (type drivers, mechanism drivers, extension drivers for example)